schemas/deviceShadow/ipShadow/ipShadow.json

```diff
@@ -0,0 +1,279 @@
+                    "$ref": "#/definitions/NetworkInfo"
+                },
+                "simInfo": {
+                    "$ref": "#/definitions/SimInfo"
+                },
+                "deviceInfo": {
+                    "$ref": "#/definitions/DeviceInfo"
+                },
+                "serviceInfo": {
+                    "$ref": "#/definitions/ServiceInfo"
+                }
+            },
+            "title": "Device"
+        },
+        "DeviceInfo": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "modemFirmware": {
+                    "type": "string"
+                },
+                "batteryVoltage": {
+                    "type": "integer"
+                },
+                "imei": {
+                    "type": "string"
+                },
+                "board": {
+                    "type": "string"
+                },
+                "appVersion": {
+                    "type": "string"
+                },
+                "appName": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "appName",
+                "appVersion",
+                "batteryVoltage",
+                "board",
+                "imei",
+                "modemFirmware"
+            ],
+            "title": "DeviceInfo"
+        },
+        "NetworkInfo": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "currentBand": {
+                    "type": "integer"
+                },
+                "supportedBands": {
+                    "type": "string"
+                },
+                "areaCode": {
+                    "type": "integer"
+                },
+                "mccmnc": {
+                    "type": "string",
+                    "format": "integer"
+                },
+                "ipAddress": {
+                    "type": "string"
+                },
+                "ueMode": {
+                    "type": "integer"
+                },
+                "cellID": {
+                    "type": "integer"
+                },
+                "networkMode": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "areaCode",
+                "cellID",
+                "currentBand",
+                "ipAddress",
+                "mccmnc",
+                "networkMode",
+                "supportedBands",
+                "ueMode"
+            ],
+            "title": "NetworkInfo"
+        },
+        "ServiceInfo": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "ui": {
+                    "type": "array",
+                    "description": "Array of services for UI",
+                    "examples": [
+                        {"fota_v1": ["HUMID", "TEMP"]}
```

Comment:
The examples should be a list of strings, because this is the item type, however, this example is a list of objects.